components/user_settings/notifications/index.ts

```diff
@@ -2,13 +2,16 @@
 // See LICENSE.txt for license information.
 
 import {connect} from 'react-redux';
-import {bindActionCreators} from 'redux';
+import {bindActionCreators, Dispatch} from 'redux';
 import {updateMe} from 'mattermost-redux/actions/users';
 import {getConfig} from 'mattermost-redux/selectors/entities/general';
 
-import UserSettingsNotifications from './user_settings_notifications.jsx';
+import {GlobalState} from 'mattermost-redux/types/store';
```

Comment:
Could you change this to import from `types/store` instead? That version of `GlobalState` extends the one from `mattermost-redux` by adding extra fields that are part of the web app.